app/views/workflow_executions/attachments/_spreadsheet_preview.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="overflow-x-auto">
+  <% data = parse_excel_file(file) %>
+  <% headers = data.first %>
+  <% rows = data[1..] %>
+  <% if data.present? %>
+    <table class="min-w-full divide-y divide-slate-200 dark:divide-slate-700">
+      <thead class="bg-slate-50 dark:bg-slate-700">
+        <tr>
+          <% headers.each do |header| %>
+            <th
+              scope="col"
+              class="
+                px-6 py-3 text-xs font-medium tracking-wider text-left uppercase text-slate-500
+                dark:text-slate-400
+              "
+            >
+              <%= header %>
+            </th>
+          <% end %>
+        </tr>
+      </thead>
+      <tbody
+        class="
+          bg-white divide-y divide-slate-200 dark:bg-slate-800 dark:divide-slate-700
+        "
+      >
+        <% rows.each do |row| %>
+          <tr>
+            <% row.each do |key, value| %>
+              <td class="px-6 py-4 text-sm whitespace-nowrap text-slate-900 dark:text-white">
+                <%= value %>
+              </td>
+            <% end %>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  <% else %>
+    <div class="text-slate-500 dark:text-slate-400">
+      "error parsing excel file"
```

Comment:
Internationalize text

Reply:
Updated in 3e0c5d83c2ac59600b4a3a01b1b3d18cbd283d80